packages/lab/src/color-chooser/ColorChooser.tsx

```diff
@@ -134,80 +134,70 @@ export const ColorChooser = ({
       onClear();
     }
   };
+
   const onTabClick = (index: number): void => {
     setActiveTab(index);
   };
 
-  const { getTriggerProps, getOverlayProps } = useOverlay({
-    placement: "bottom",
-    open,
-    onOpenChange: handleOpenChange,
-  });
-
-  return (
-    <>
+  const OverlayContent = () => (
+    <div
+      className={clsx(withBaseName("overlayContent"))}
+      data-testid="overlay-content"
+    >
       <Button
-        {...getTriggerProps<typeof Button>({
-          className: clsx(withBaseName("overlayButton"), {
-            [withBaseName("overlayButtonHiddenLabel")]: hideLabel,
-          }),
-          // @ts-ignore
-          "data-testid": "color-chooser-overlay-button",
-          disabled: readOnly,
-          ...buttonProps,
-        })}
+        data-testid="default-button"
+        variant="secondary"
+        className={clsx(withBaseName("defaultButton"))}
+        onClick={onDefaultSelected}
       >
-        {color && (
-          <div
-            className={clsx(withBaseName("overlayButtonSwatch"), {
-              [withBaseName("overlayButtonSwatchWithBorder")]:
-                color?.hex.startsWith("#ffffff"),
-              [withBaseName("overlayButtonSwatchTransparent")]: isTransparent(
-                color?.hex
-              ),
-            })}
-            style={{
-              backgroundColor: color?.hex,
-            }}
-          />
-        )}
-        {!hideLabel && (
-          <div className={withBaseName("overlayButtonText")}>
-            {displayColorName ?? placeholder ?? "No color selected"}
-          </div>
-        )}
+        <RefreshIcon className={clsx(withBaseName("refreshIcon"))} />
+        Default
       </Button>
-      <Overlay
-        {...getOverlayProps({
-          adaExceptions: {
-            showClose: false,
-          },
+      <DictTabs
+        tabs={tabsMapping}
+        hexValue={color?.hex}
+        onTabClick={onTabClick}
+        activeTab={activeTab}
+      />
+    </div>
+  );
+
+  return (
+    <Overlay placement="bottom" data-testid="color-chooser-overlay">
+      <OverlayTrigger>
```

Comment:
could the OverlayTrigger not be a button by default? then you avoid children and would not need to clone but just pass down ref and onClick

Reply:
updated